parm/atm/obs/config/sfc.yaml

```diff
@@ -1,14 +1,14 @@
 obs space:
```

Comment:
Is there a difference in purposes between the sfc.yaml in the config directory and the sfc.yaml in the testing directory?

Reply:
the intention was for config/ to be the 'production' ready files, as in they have been tested and work, whereas testing/ might not yet work properly but are there as examples/a work in progress. But I'm open to changing/removing that.

Reply:
The input formats of the YAMLs in the UFO tests are different than the 'real' executables, so that was why I split them into separate directories.